apps/cb-ceiling/test/daily_model_cycle_station_query.test.js

```diff
@@ -34,8 +34,13 @@ describe('dieoff_query', () => {
                      AND NOT odata.Ceiling < 3000.0 THEN '1' ELSE '0' END || ';' || CASE WHEN NOT m0data.Ceiling < 3000.0
                      AND odata.Ceiling < 3000.0 THEN '1' ELSE '0' END || ';' || CASE WHEN NOT m0data.Ceiling < 3000.0
                      AND NOT odata.Ceiling < 3000.0 THEN '1' ELSE '0' END) AS sub_data
+<<<<<<< HEAD
+      from vxDBTARGET  AS m0
+          JOIN mdata AS o ON o.fcstValidEpoch = m0.fcstValidEpoch
+=======
       FROM vxdata._default.METAR AS m0
           JOIN vxdata._default.METAR AS o ON o.fcstValidEpoch = m0.fcstValidEpoch
+>>>>>>> development
```

Comment:
The tests in `apps/cb-ceiling/test` appear to have these unresolved git merge comments in them. We'll want to hand resolve them since they aren't valid JS. (Or SQL statements)

Reply:
Good catch @ian-noaa , these looks like old tests based on our original data model. I will delete them.